src/photos/src/NIPhotoScrollView.m

```diff
@@ -507,4 +470,36 @@ - (void)setFrameAndMaintainState:(CGRect)frame {
   [_scrollView setNeedsLayout];
```

Comment:
Previously this call to -setNeedsLayout would result in the centering logic of NICenteringScrollView being triggered. With the changes in this PR that will no longer happen. Is that a problem? If not, why is it not a problem?

I would imagine that it's not a problem because we have just set the frame and thus we can expect that the new centering logic in -layoutSubviews will be triggered? If that is the case, is it still necessary to call -setNeedsLayout on the child scroll view?

Reply:
Good point, it is no longer needed for the reason you wrote.